pkgs/build-support/dotnet/build-dotnet-global-tool/update.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=./. -i bash -p curl jq nix common-updater-scripts
+# shellcheck shell=bash
+
+set -euo pipefail
+
+attr=$UPDATE_NIX_ATTR_PATH
+
+nixeval() {
+    nix --extra-experimental-features nix-command eval --json --impure -f . "$1" | jq -r .
+}
+
+nugetName=$(nixeval "$attr.nupkg.pname")
+
+# always skip prerelease versions for now
+version=$(curl -fsSL "https://api.nuget.org/v3-flatcontainer/$nugetName/index.json" |
+    jq -er '.versions | last(.[] | select(match("^[0-9]+\\.[0-9]+\\.[0-9]+$")))')
```

Comment:
Do we have a guarantee these will always be ordered from oldest to newest version (according to semver)?

Reply:
https://learn.microsoft.com/en-us/nuget/api/package-base-address-resource#enumerate-package-versions

I can't find any specification of the order.

In practice they seem to be ordered by version and not e.g. by published date.

Reply:
Fair, a bit uneasy we're depending on undocumented behavior but I don't think there's an easy way to do it without using another language.